src/datumaro/util/mask_tools.py

```diff
@@ -383,9 +416,30 @@ def to_uncompressed_rle(rle: Rle, *, width: int, height: int) -> UncompressedRle
     if is_uncompressed_rle(rle):
         return rle
 
-    from pycocotools import mask as mask_utils
+    return pycocotools_mask.frPyObjects(rle, height, width)
+
+
+def mask_to_bboxes(mask: BinaryMask) -> List[List[int]]:
```

Comment:
```suggestion
def mask_to_bboxes(mask: BinaryMask) -> List[BboxIntCoords]:
```

I think it will be better to remove `BboxCoords` from this file and replace uses with `BboxIntCoords`, or at least create an alias

Reply:
Replaced List[int] with BboxIntCoords. I dont get why it is better to remove `BboxCoords` from this file

Reply:
`BboxCoords` is, basically, the same as `BboxIntCoords`, added by different teams. There are not many uses for `BboxCoords` now, so it makes sense to replace it with more used `BboxIntCoords`.

Reply:
done